components/layouts/UnauthenticatedLayout/index.js

```diff
@@ -0,0 +1,23 @@
+import { Layout } from "antd";
+
+import styles from "./style.module.css";
+
+function UnauthenticatedLayout({ children }) {
+  const { Content } = Layout;
+  return (
+    <Layout>
+      <Content className={styles.content}>
+        <div className={styles.main}>
+          <img
+            className={styles.logo}
+            src="/img/logo.svg"
+            alt="CoderDojo Logo"
+          />
+          <div className={styles.form}>{children}</div>
+        </div>
+      </Content>
+    </Layout>
+  );
+}
+
+export default UnauthenticatedLayout;
```

Comment:
I think this name should be something like `AuthenticationLayout`, because components should not reveal user state. This layout is not "Unauthenticated" but rather were you could deal with "Authentication" related actions, like sign up or login.